include/pci_caps/px.h

```diff
@@ -163,11 +163,12 @@ struct pxdcap2 {
     uint32_t nprpr:1;
     uint32_t ltrs:1;
     uint32_t tphcs:2;
+    uint32_t res1:4;
```

Comment:
Thanks, good spot!

But these bits are actually defined in pci express:

LN System CLS (bits 14, 15) ("undefined" in pciex6!)
10-bit Tag Completer Supported (bit 16)
10-bit Tag Requester Supported (bit 17)


as well as some later bits:

Emergency Power Reduction Supported (bits 24,25)
Emergency Power Reduction Initialization Required (bit 26)
DMWr completer supported (bit 28)
DMWr lengths supported (bit 29, 30)
FRS Supported (bit 31)


Could I talk you into updating this struct to reflect these?